cmd/atlas/inspect.go

```diff
@@ -12,16 +16,17 @@ var (
 	inspectCmd = &cobra.Command{
 		Use:   "inspect",
 		Short: "Inspect atlas schema.",
-		Run:   func(cmd *cobra.Command, args []string) {},
+		Run:   func(cmd *cobra.Command, args []string) { runInspect(inspectFlags.dsn) },
 		Example: `
-atlas schema inspect -d mysql://user:pass@host:port/dbname
-atlas schema inspect -d postgres://user:pass@host:port/dbname
-atlas schema inspect --dsn sqlite3://path/to/dbname.sqlite3`,
+atlas schema inspect -d "mysql://user:pass@host:port/dbname"
+atlas schema inspect -d "postgres://user:pass@host:port/dbname"
+atlas schema inspect --dsn "sqlite3://path/to/dbname.sqlite3"`,
 	}
 )
 
 func init() {
 	schemaCmd.AddCommand(inspectCmd)
+	schemaCmd.SetOut(os.Stdout)
```

Comment:
What's the benefit of going through cobra and coupling ourselves to it instead of just writing to stdout? 
Remember, complexity comes from two things: 1. coupling 2. obscurity - here you get both

Reply:
I'll move it to a printer module but it's important to make sure what goes via std error and what in stdout. by using cobra the cli is consistent. (think about users doing atlas something something > atlas.hcl